### ISSUE TITLE
Typescript path aliases cannot be used

### ISSUE THREAD

**Issue by FredyC** [mattpocock/xstate-codegen#23]
I am using path aliases quite heavily. An example of my `tsconfig.json`.

```json
{
  "compilerOptions": {
    "baseUrl": "src",
    "paths": {
      "~machinery/graphql": ["generated/machinerygraphql"],
      "~machinery/*": ["*"],
      "~tools/*": ["../../tools/src/*"]
    },
  },
}
```

Then in the code I can do eg. `import { getNow } from '~tools/timing'`. Works like a charm with other tooling (after some hassle).

I suppose based on the https://github.com/mattpocock/xstate-codegen/issues/21#issuecomment-691312065 it should be solved when transition is done?

**Comment by mattpocock** [comment]
This makes sense why this wouldn't work, rollup doesn't know about your aliases. Another reason why the transition mentioned in #21 makes sense.

As with that comment, a workaround is to extract the machines into their own file which doesn't use aliases. Is that workable for you?

**Comment by FredyC** [comment]
> workaround is to extract the machines into their own file which doesn't use aliases. Is that workable for you?

Not really, because those imported functions are an inherent part of the machine. I am even used to splitting everything like this and the main `bootMachine.ts` does not have any aliased imports.

![image](https://user-images.githubusercontent.com/1096340/92994122-c22c3d00-f4f7-11ea-9a6a-8900eae94d29.png)

It's alright, I will wait for the transition. You are doing a great thing here. Looking forward to a moment when all these nasty edge cases are figured out.
